lib/core/src/server/preview/entries.js

```diff
@@ -57,13 +57,18 @@ export async function createPreviewEntry(options) {
   if (stories && stories.length) {
     entries.push(path.resolve(path.join(configDir, `generated-stories-entry.js`)));
 
-    const files = (await Promise.all(stories.map((g) => glob(g)))).reduce((a, b) => a.concat(b));
+    const files = (
+      await Promise.all(stories.map((g) => glob(path.join(configDir, g))))
```

Comment:
@ndelangen Is this correct? I don't think it makes sense to join the glob with the configDir. This means that all globs before this pr won't work anymore

Reply:
yes this is intentional indeed.

Reply:
@ndelangen if this is a breaking change do we need to document it in `MIGRATION.md`?

Reply:
This is def a breaking change that should be documented. But it seems really odd to me that your glob become `.storybook/**/*.stories`. IDK when I would ever want that. In reality it always must be something like `.storybook/../**/*.stories`.